new/openstack-manager-blueprint.yaml

```diff
@@ -426,6 +426,14 @@ node_templates:
         policy_engine:
           start_timeout: 30
 
+        import_resolver:
```

Comment:
please add documentation explaining what this means... including the fact that it fails over to the online files, etc..